pages/treeherder.py

```diff
@@ -7,17 +7,25 @@
 from selenium.webdriver.support.ui import WebDriverWait as Wait
 
 import expected
+import random
+
 from pages.base import Base
 from pages.page import Page
 
 
 class TreeherderPage(Base):
 
+    _active_watched_repo = (By.CSS_SELECTOR, '#watched-repo-navbar button.active')
     _first_resultset_datestamp_locator = (By.CSS_SELECTOR, '.result-set .result-set-title-left > span a')
```

Comment:
I see that this is probably due to the unintentional removal of `open_single_resultset` so it should stay.